### DISCUSSION TITLE
Linux - Plotting Log

### DISCUSSION THREAD

**Discussion by pieterhelsen** [Chia-Network/chia-blockchain#1631] [Q&A]
Hi,

Could anyone point me in the direction of the plotting log on Linux (when installed via Virtual Environment)?
I've looked in the usual places (~/.chia/mainnet and ~/chia-blockchain) using the `find` command, but couldn't find anything resembling the plotter_log txt files on Windows. 

The `~/.chia/mainnet/plotter` directory doesn't exist, while all others (wallet, run, debug, ...) do. 

Any help would be greatly appreciated!

**Comment by hoffmang9** [comment]
Unless you are using the GUI on a Linux Desktop version, capturing the output of the plotting process on the CLI is up to you.

**Reply by pieterhelsen** [reply]
Ahhh, that makes sense! Thanks Gene.

**Reply by djdookie** [reply]
I just suggested that the CLI should also write plotter logs autonomously without the need to manually capture CLI output:
https://github.com/Chia-Network/chia-blockchain/issues/3530

**Comment by Cyrus8** [comment]
I have this directory : ~/.chia/mainnet/log  but it is empty. 
The output of the plots command itself gives me some informations as "Starting phase 2/4: Backpropagation into tmp files... Tue May  4 08:03:01 2021" but as it is my first plot I have no idea about the total duration needed for each phase. 
Is there some more detailed logs?  Is each phase of the generation supposed to take the same amount of time?

**Comment by justinphelps** [comment]
You'll have to capture the plotter logs yourself. The log destination for plotting is not currently configurable.
If you want to do it yourself, you can do something like:
chia plots create -t /chia/tmpdir -d /chia/plots **2>> ~/logs/mylog.txt**
Alternatively you could try using a plotting management tool like [plotman](https://github.com/ericaltendorf/plotman).

**Reply by tarchive** [reply]
> chia plots create -t /chia/tmpdir -d /chia/plots 2>> ~/logs/mylog.txt

Your example command does not work. Only the first few lines go to the log file and the rest after `Starting plot 1/1` still gets output to the terminal.

In my case i would like it on the terminal and log file so i ended up using 
`chia plots create -t /chia/tmpdir -d /chia/plots 2>&1 | tee ~/logs/mylog.txt`
